Makefile

```diff
@@ -27,4 +27,4 @@ endif
 .PHONY: md-lint
 md-lint: ## Lint markdown files
 	$(ECHO_PREFIX) printf "  %-12s ./...\n" "[MD LINT]"
-	$(CMD_PREFIX) podman run --rm -v $(CURDIR):/workdir --security-opt label=disable docker.io/davidanson/markdownlint-cli2:v0.12.1 > /dev/null
+	$(CMD_PREFIX) podman run --rm -v $(CURDIR):/workdir --security-opt label=disable docker.io/davidanson/markdownlint-cli2:latest > /dev/null
```

Comment:
Having the latest here seems worse than the version pinning.

Reply:
This makefile is not part of the action execution. It is a convenience for a local dev to run markdownlint. I have the `markdownline-cli2` command installed, so I can just run it directly and not use a container. 

Since the makefile is not a workflow file, dependabot will not keep the version up-to-date like in the workflow file. Since this file is not part of the action exection, using `latest` is a modest compromise since it will roughly kept in sync with the version in the workflows which is kept up-to-date by dependabot.

Reply:
Noted, thanks for that clarification.